src/api.ts

```diff
@@ -84,6 +84,11 @@ export interface Request {
 	 * The external identifier for the release.
 	 */
 	commit: string;
+
+	/**
+	 * Mark the created release as a draft
+	 */
+	draft: boolean;
```

Comment:
```suggestion
	draft?: boolean;
```
this has a default value, so we probably don't need to suddenly require it in the typings. Also allows us to bump balena-release in the CLI and not get any errors even if we never care to provide that option to users.

Reply:
Yes, you are correct. I'll make the change